.github/workflows/appmap-analysis.yml

```diff
@@ -13,7 +13,24 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           bundler-cache: true
+      - name: Enable appmap gem update
+        run: bundle config unset deployment    
+      - name: Install AppMap tools
+        id: install-appmap
+        uses: getappmap/install-action@v1
+        with:
+          project-type: bundler
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v9
+        with:
```

Comment:
At this point in the build, all changes to the repo should be committed. That will ensure that the .gitignore, Gemfile, and Gemfile.lock are committed. So you should be able to remove the `with:` option.